localstack/utils/bootstrap.py

```diff
@@ -613,27 +614,43 @@ def configure_container(container: LocalstackContainer):
         if value is not None:
             container.env_vars[env_var] = value
     container.env_vars["DOCKER_HOST"] = f"unix://{config.DOCKER_SOCK}"
-    container.env_vars["HOST_TMP_FOLDER"] = config.HOST_TMP_FOLDER
+    container.env_vars["HOST_TMP_FOLDER"] = config.dirs.shared_tmp
 
     # TODO discuss if this should be the default?
     # to activate proper signal handling
     container.env_vars["SET_TERM_HANDLER"] = "1"
 
-    # data_dir mounting and environment variables
-    bind_mounts = []
-    data_dir = os.environ.get("DATA_DIR", None)
-    if data_dir is not None:
-        container_data_dir = "/tmp/localstack_data"
-        container.volumes.add((data_dir, container_data_dir))
-        container.env_vars["DATA_DIR"] = container_data_dir
+    configure_volume_mounts(container)
 
-    # default bind mounts
-    container.volumes.add((config.TMP_FOLDER, "/tmp/localstack"))
-    bind_mounts.append((config.DOCKER_SOCK, config.DOCKER_SOCK))
+    # mount docker socket
+    container.volumes.append((config.DOCKER_SOCK, config.DOCKER_SOCK))
 
     container.additional_flags.append("--privileged")
 
 
+def configure_volume_mounts(container: LocalstackContainer):
+    source_dirs = config.dirs
+    target_dirs = Directories.for_container()
+
+    # default shared directories
+    for name in ["var", "cache", "init", "logs"]:
```

Comment:
nit: Maybe we could define this list as string constants (or a static list of strings) on the `Directories` class. (to maintain the folder names only in a single place)